### ISSUE TITLE
Config resets itself

### ISSUE THREAD

**Issue by Darunia18** [IDragonfire/DragonAntiPvPLeaver#36]
Whenever I change something in the config file and reload the plugin, the config file resets to the default.

**Comment by IDragonfire** [comment]
yeah .. the plugin save the config on shutdown and overwrite your changes ...
